src/target/detection.c

```diff
@@ -290,7 +279,7 @@ void side_sensors_calibration(void)
 	for (i = 0; i < SIDE_CALIBRATION_READINGS; i++) {
 		left_temp += distance[SENSOR_SIDE_LEFT_ID];
 		right_temp += distance[SENSOR_SIDE_RIGHT_ID];
-		sleep_ticks(SENSORS_SM_TICKS);
+		sleep_ticks(2);
```

Comment:
why did you change it to 2? Coudl you add an explicative macro?

Reply:
How would you call it? It is just some small delay in between readings, it could be any other number really.

I could create a `SIDE_CALIBRATION_READINGS_DELAY_TICKS` but looks huge just for one use and it really is not important. Does it hurt to much in your eyes? :joy: